scripts/train.py

```
@@ -591,6 +725,58 @@ def parse_args():
+        default=None,
+        help="Override the drafter's num_key_value_heads.",
+    )
+    parser.add_argument(
+        "--draft-head-dim",
+        type=int,
+        default=None,
+        help="Override the drafter's per-head hidden dimension.",
+    )
+    parser.add_argument(
+        "--draft-rope-scaling",
+        type=lambda s: json.loads(s) if s else None,
```

Ruff (0.15.15)

[error] Undefined name `json`

(F821)